configs/opencv-ovms/scripts/run_gst.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# Copyright (C) 2023 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+echo "Run gst pipeline"
+rmDocker=--rm
+if [ ! -z "$DEBUG" ]
+then
+	# when there is non-empty DEBUG env, the output of app outputs to the console for easily debugging
+	rmDocker=""
+fi
+
+echo $rmDocker
+TAG=sco-soc:2.0
+pipeline="yolov5s.sh"
+
+echo $PLATFORM
+
+bash_cmd="/home/pipeline-server/framework-pipelines/$PLATFORM/$pipeline"
+if [ "$STREAM_DENSITY_MODE" == 1 ]; then
+	echo "Starting Stream Density"
+	bash_cmd="./stream_density_framework-pipelines.sh framework-pipelines/$PLATFORM/$pipeline"
```

Comment:
we need to discuss this: if you are using ovms client to launch a profile, the stream density is actually launching by ovms-client not by this. so this command doesn't work for stream density mode.

Reply:
That a great point. For now I can remove it and we can use this only for the normal case. We can fix the stream density when we move the cmd_client to the new architecture.